test/test-zero-len-file.sh

```diff
@@ -50,7 +50,8 @@ begin_test "pull zero len file"
   clone_repo "$reponame" clone
   rm clone.log
 
-  git status | grep "working directory clean"
+  git status 2>&1 | tee status.log
+  grep "working (directory|tree) clean" status.log
```

Comment:
`grep` doesn't work on that pattern for me, it had to be `egrep`. This is what I used in #1370